lib/recording/RecordingManager.js

```diff
@@ -43,119 +44,136 @@ class RecordingManager {
 
         logger.info("Initializing recording manager");
 
-        setInterval(()=>{
+        setInterval(()=> {
             this.handleRecordingEntries()
-        }, recordingTimeInterval, this)
+        }, recordingTimeInterval, this);
     }
 
     startRecording(entryObj, flavorsObjArray){
         return Q.resolve().then(()=> {
-
-            function makeFlavorDirectories() {
-                logger.debug("[%s-%s] About to create directories %s", entryObj.entryId, entryObj.recordedEntryId, recordingEntrySession.flavors)
+            function makeFlavorDirectories(recordingEntrySession) {
+                logger.debug(`[${entryObj.entryId}-${entryObj.recordedEntryId}] About to create directories ${recordingEntrySession.flavors}`);
                 let creatingDirectoriesPromise = _.map(recordingEntrySession.flavors, (flavorId)=> {
                     let flavorPath = path.join(recordingEntrySession.recordingSessionPath, flavorId)
                     return qio.makeDirectory(flavorPath)
-                })
+                });
                 return Q.allSettled(creatingDirectoriesPromise)
                     .then((results)=> {
                         _.each(results, (promise)=> {
                             if (promise.state === 'rejected') {
                                 if (promise.reason && promise.reason.code === 'EEXIST') {
-                                    logger.debug("[%s-%s] path %s already exist", entryObj.entryId, entryObj.recordedEntryId, promise.reason.path)
+                                    logger.debug(`[${entryObj.entryId}-${entryObj.recordedEntryId}] path ${promise.reason.path} already exist`);
                                 }
                                 else {
-                                    logger.error("[%s-%s] Failed to make directory %s", entryObj.entryId, entryObj.recordedEntryId, ErrorUtils.error2string(promise.reason))
+                                    logger.error(`[${entryObj.entryId}-${entryObj.recordedEntryId}] Failed to make directory ${ErrorUtils.error2string(promise.reason)}`);
                                 }
                             }
                         })
                     })
             }
-            const getRecordingEntrySession = ()=>{
-                let recordingEntrySession = null
+
+            function getRecordingEntrySession(entryObj) {
                 let entryId = entryObj.entryId;
```

Comment:
I pass the function entryObj as a parameter, which is everything needed for it to run.
Also, tested it and nothing got broken. If still want me to change it be let me know